translate/src/modules/user/components/UserMenu.css

```diff
@@ -106,3 +106,72 @@
   margin: 5px 0;
   padding: 0;
 }
+
+/* Profile menu */
+
+.toggle-button {
+  font-size: 0;
+  cursor: default;
+  padding: 4px 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: 8px;
+}
+
+.toggle-button:hover {
+  background: transparent;
+  color: inherit;
+}
+
+.toggle-button .help {
+  padding-bottom: 5px;
+  font-size: 12px;
+  text-transform: uppercase;
+}
+
+.toggle-button button {
+  background: var(--black-3);
+  border: 1px solid var(--light-grey-3);
+  border-radius: 3px;
+  box-sizing: border-box;
+  color: var(--grey-6);
+  font-size: 14px;
+  font-weight: 100;
+  padding: 4px;
+  width: 78px;
+  margin: 0;
+  float: none;
+}
+
+.toggle-button button:first-child {
+  border-right-color: var(--grey-3);
+  border-top-right-radius: 0;
+  border-bottom-right-radius: 0;
+}
+
+.toggle-button button:nth-child(2) {
+  margin: 0 8px;
+}
+
+.toggle-button button:last-child {
+  border-left: 1px solid var(--light-grey-3);
+  border-top-left-radius: 0;
+  border-bottom-left-radius: 0;
+}
+
+.toggle-button button:hover {
+  color: var(--light-grey-6);
+}
+
+.toggle-button button.active {
+  background: var(--dark-grey-1);
+  border-color: var(--grey-3);
+  color: var(--light-grey-7);
+  font-weight: 400;
+}
```

Comment:
We'll need to sync these colors with the ones used in https://github.com/mozilla/pontoon/pull/2997/files#diff-30336dc4648b06dd8f0228e1c1013ab8260d8a5404cb86cba8187ed141519f1bR1461, but that should be done in the theme CSS PR.